charts/wildfly/values.yaml

```diff
@@ -765,7 +765,7 @@ wildfly:
     image:
       registry: docker.io
       repository: bitnami/os-shell
-      tag: 12-debian-12-r27
+      tag: 12-debian-12-r28
```

Comment:
@yvesjaggi one of the reasons to *not* put here the full upstream _values.yaml_. Cf. https://github.com/CHORUS-TRE/chorus-tre/pull/131#discussion_r1729132021